Source/monster.cpp

```diff
@@ -2070,7 +2070,42 @@ void AiAvoidance(int i, bool special)
 	monster.CheckStandAnimationIsLoaded(md);
 }
 
-void AiRanged(int i, missile_id missileType, bool special)
+ missile_id GetMissileType(_mai_id ai)
+{
+	switch (ai) {
+	case AI_GOATMC:
+		return MIS_ARROW;
+		break;

```

Comment:
you don't need breaks after return

Reply:
isn't that good practice or something ? :D

Reply:
Not after a return